lab-mikeporter/test/lib/mockUsers.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const faker = require('faker');
+const User = require('../../model/user.js');
+
+const mockUser = module.exports = {};
+
+mockUser.one = () => {
+  return new User({
+    name: faker.name.firstName(2),
+    city: faker.address.city(),
+    weight: faker.random.number(),
+  })
+    .save();
+};
+
+mockUser.many = (num) => {
+  let mockUserArray = new Array(num)
+    .fill(0).map(() => mockUser.one());
```

Comment:
get used to object/array manipulation for working with different data! very very very common to work with responses and grabbing/displaying certain keys/values